pyshacl/cli.py

```diff
@@ -211,13 +216,33 @@ def main(prog: Union[str, None] = None) -> None:
     if str_is_true(do_server) or args.server:
         from pyshacl.sh_http import main as http_main
 
+        # http_main calls sys.exit(0) and never returns
         http_main()
-    elif not args.data:
+    if not args.data:
         # No datafile give, and not starting in server mode.
-        sys.stderr.write('Validation Error. No DataGraph file supplied.\n')
+        sys.stderr.write('Validation Error. No DataGraph file or endpoint supplied.\n')
```

Comment:
Can you use a Values Error or Input Error  instead of Validaton Error? The latter should be reserved for validation results only.